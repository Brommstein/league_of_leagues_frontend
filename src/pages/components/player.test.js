import React from 'react';
import Player from './player';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('Player Component', () => {
    const props = {
        players: {
            userid: 56,
            leaguename: 'ComponentTest',
            preferedrole: 'jungle',
            secondaryrole: 'adc',
            team: 'CTT'
        }
    }
    it('component', () => {
        const playerComponent = shallow(<Player {...props} />);
        expect(playerComponent).toMatchSnapshot();
    });
});