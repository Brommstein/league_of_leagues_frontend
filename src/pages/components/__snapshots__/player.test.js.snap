// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player Component component 1`] = `ShallowWrapper {}`;
